src/jquery.multiselect.js

```diff
@@ -122,7 +122,7 @@
       // create a unique namespace for events that the widget
       // factory cannot unbind automatically. Use eventNamespace if on
       // jQuery UI 1.9+, and otherwise fallback to a custom string.
-      this._namespaceID = this.eventNamespace || ('multiselect' + multiselectID);
+      this._namespaceID = this.eventNamespace.slice(1) || ('multiselect' + multiselectID);
```

Comment:
This mentions that the second condition is for jQuery UI prior to 1.9, can probably just delete it since the widget now requires 1.11.